.travis.yml

```diff
@@ -1,20 +1,15 @@
-sudo: false
+sudo: required
+dist: trusty
+
 language: scala
 jdk:
   - oraclejdk8
 scala:
   - 2.12.3
-cache:
```

Comment:
why did we remove the cache?

Reply:
We don't benefit from caching since the cache is downloaded from S3 (https://docs.travis-ci.com/user/caching#How-does-the-caching-work%3F). For sbt it might makes sense since it's rebuilt each time. I wanted to simplify things.